provisioning/datasources/x-ray.e2e.yaml

```diff
@@ -0,0 +1,19 @@
+# config file version
+apiVersion: 1
+
+deleteDatasources:
+  - name: AWS X-Ray E2E
+    orgId: 1
+
+datasources:
+  - name: AWS X-Ray E2E
+    type: grafana-x-ray-datasource
+    uid: x-ray-e2e
+    access: proxy
+    jsonData:
+      authType: keys
+      defaultRegion: us-east-2
+    secureJsonData:
+      accessKey: ${AWS_ACCESS_KEY_ID}
```

Comment:
How come we're not pulling the creds from the plugin provisioning repo? I can see how this way it's easier, at least for x-ray, since it needs only key ID and secret key, and we don't have to deal with permissions for the github action. Is this why?
Is this how we will provision e2e tests from now on, or will it be a decision on datasource-to-datasource basis, depending on how much data is required to provision?

Reply:
Yes. Also I think we should probably use Vault for these things to pull in the minimal amount of information needed to provision an e2e test DS.